### DISCUSSION TITLE
Clean URLs

### DISCUSSION THREAD

**Discussion by jcehenne** [Pagefind/pagefind#280] [Q&A] [answered]
Is it possible to omit the extension .html into the search index?

**Comment by bglw** [comment, marked as answer]
Hey @jcehenne 👋 

The best way to control URLs it to tag a custom URL on your pages — see the docs for this here: https://pagefind.app/docs/ui/#overriding-the-url-of-a-result

Alternatively you could use the [processResult hook](https://pagefind.app/docs/ui/#process-result) of the UI to tidy results on the fly

**Reply by despens** [reply]
For reference, the documentation for overriding the URL of a result has moved here: https://pagefind.app/docs/ui-usage/#overriding-the-url-of-a-result
